crates/db_views/src/comment_view.rs

```diff
@@ -779,6 +795,35 @@ mod tests {
     cleanup(data, pool).await;
   }
 
+  #[tokio::test]
+  #[serial]
+  async fn test_creator_is_moderator() {
+    let pool = &build_db_pool_for_tests().await;
+    let pool = &mut pool.into();
+    let data = init_data(pool).await;
+
+    // Make one of the inserted persons a moderator
+    let person_id = data.inserted_person_2.id;
+    let community_id = data.inserted_community.id;
+    let form = CommunityModeratorForm {
+      community_id,
+      person_id,
+    };
+    CommunityModerator::join(pool, &form).await.unwrap();
+
+    // Make sure that they come back as a mod in the list
+    let comments = CommentQuery {
+      ..Default::default()
+    }
+    .list(pool)
+    .await
+    .unwrap();
+
+    assert!(comments[1].creator_is_moderator);
```

Comment:
You can also add a check that `creator_is_moderator` is false in one of the other test cases.

Reply:
That's done in the other tests in L857